### PR TITLE
[FIX] PWC - Setup killpin/powerhold ASAP or printer suicides

### PR DESCRIPTION
**Pull request by Rockman18**
### Description

This PR fix the issue detected during MKS_WIFI_MODULE release (#20191). With a PWC the printer suicides during the serial init of the MKS wifi. The setup of killpin/powerhold occurs too late in the process.

### Requirements

- MKS PWC
- MKS WIFI MODULE

### Benefits

The printer doesn't suicide during init.

### Configurations

N/A

### Related Issues

https://github.com/MarlinFirmware/Marlin/pull/20191#issuecomment-753000214

### CODE REVIEW

**Comment by X-Ryl669** [comment]
Did you test this ? (I don't have the configuration to test it). 
It looks good to me.

File: Marlin/src/MarlinCore.cpp [hunk 1]
```diff
@@ -983,6 +983,10 @@ void setup() {
   #endif
   #define SETUP_RUN(C) do{ SETUP_LOG(STRINGIFY(C)); C; }while(0)
 
+  SETUP_RUN(setup_killpin()); // Must be set early or the printer will suicide
```
Comment (X-Ryl669) [inline Marlin/src/MarlinCore.cpp]: If you are combining both, make the comment unique and above both line.

**Comment by Rockman18** [comment]
Yes i've tested it. It fixed my issue. I don't know but I wonder if only powerhold would be enough.

**Comment by Fighter777** [comment]
hello, now the psu pin on SKR pro 1.1 don't turn on -_-"

https://github.com/MarlinFirmware/Marlin/issues/21046

Before :
1) PSU pin on
2) Screen display : Switch Power Off
image


After :
1) PSU pin off
2) Screen display : Switch Power Off

**Comment by Evg33** [comment]
https://github.com/MarlinFirmware/Marlin/issues/21182

**Pull request merged**